app/scripts/views/settings/avatar_change.js

```diff
@@ -45,15 +45,17 @@ define(function (require, exports, module) {
       }
     },
 
+    context: function () {
+      var account = this.getSignedInAccount();
```

Comment:
There may be some subtle avatar cache problems here. If the user signs in for the first time with an account that has an avatar, in theory the avatar may not be set by this point. `displayAccountProfileImage` fetches the profile. The reverse problem is also possible. If the user is signed in here, has a profile set, and removes the profile photo on another device, the clear button will be displayed.

I think in reality, this shouldn't be much of a problem if the user hits `/settings` first. Some testing is needed.